### ISSUE TITLE
Reproducing results in ICCV '21 paper

### ISSUE THREAD

**Issue by sumith1896** [anindita127/Complextext2animation#1]
Reposting from the email thread as requested. Thank you for releasing the code w/ instructions!. I had a few questions:

1) I was able to sample motions using the pre-trained model and also run the APE metric script. It gives out 20 values. How do I compute the 4 values reported in the paper in Figure 4 (root, global, mean w/o traj and w/ traj) from them?
2) I wanted to also generate GT renderings, do you know what command I need to use?
3) Finally, it would also be really useful if I could also get the number for the baselines you report. I see that you have eval_APE_baseline scripts, could you let me know how to use them? I see that the codebase is based on language2pose but the train commands in the original repo do not seem to work (neither does using their pre-trained model due to torch version mismatch).

Thanks in advance!

**Comment by anindita127** [comment]
Thanks for your interest in our work. I'll answer the questions as detailed as possible.

1. You can get the mean values from Table 1 in our supplementary materials (https://openaccess.thecvf.com/content/ICCV2021/supplemental/Ghosh_Synthesis_of_Compositional_ICCV_2021_supplemental.pdf).  You will already find the root trajectory, pelvis, torso and hips directly from the eval_APE.py code. As for the arms we have added the loss of shoulder, elbow and wrist. For legs we have added the error terms of knee, ankle and foot. And for neck we have added the loss terms of upper neck and lower neck.
2. You can find the code to render groundtruth in the language2pose repo (https://github.com/chahuja/language2pose#rendering-ground-truths)
3. You can run:   python eval_APE_baseline.py -load <path_to_language2pose_pretrained_models>
You have to download the pre-trained models of language2pose from https://github.com/chahuja/language2pose#pre-trained-models I haven't trained the language2pose model myself, only used their pre-trained model for evaluation.

Hope these answers your questions.
